test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -59,15 +60,15 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
     val declarationWithId = declaration.copy(declarationId = declarationId)
     server
       .stubFor(
-        get(urlPathEqualTo(s"$declarationsUrl/${declarationWithId.declarationId.value}"))
+        get(urlPathEqualTo(s"${config.mibDeclarationsUrl}/${declarationWithId.declarationId.value}"))
```

Comment:
This can just be `get(urlPathEqualTo(s"/declare-commercial-goods/declarations/${declarationWithId.declarationId.value}"))`